sql-plugin/src/main/spark350emr/scala/com/nvidia/spark/rapids/GpuOutputAdapterExec.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Notice that the emr/pom.xml is a new file in this PR with NVIDIA copyrights but this file does not have the copyright -- intentional?

Reply:
I missed the copyright. Will add it.